packages/frontend-core/src/components/grid/stores/clipboard.ts

```diff
@@ -1,10 +1,41 @@
-import { derived, writable, get } from "svelte/store"
+import { derived, writable, get, Writable, Readable } from "svelte/store"
 import { Helpers } from "@budibase/bbui"
 import { parseCellID, getCellID } from "../lib/utils"
 import { NewRowID } from "../lib/constants"
+import { Store as StoreContext } from "."
 
-export const createStores = () => {
-  const clipboard = writable({
+type ClipboardStoreData =
+  | {
+      value: string[][]
```

Comment:
`value` here can be any cell value, so it doesn't need to just be a string. We could be specific about the range possible types but it can be any data type that we support, so `any` is probably appropriate.

Reply:
I thought that the value would always be stringified. Updated to any